import-export-cli/box/resources/kubernetes_resources/api_params_cm.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:
+data:
```

Comment:
Here as well. Shall we keep a new line?

Reply:
Fixed with 60ffa8adb24316d4d7b9fcbdc61476b31af7f787